loadtest/user/user.go

```diff
@@ -25,6 +26,7 @@ type User interface {
 	SignUp(email, username, password string) error
 	Login() error
 	Logout() (bool, error)
+	CreatePost(post *model.Post) error
```

Comment:
I am updating the Id of the post that is passed by pointer, so that information is already available

Reply:
That's an update by side effect which I would like to avoid. I wouldn't rely too much on passing pointers especially since we are trying to be immutable on the controller side. I think returning the post id is a better option.